### PR TITLE
FEATURE: NAS-232 Add explainAfm download resource

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: NAS-232

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by Tulean** [comment]
extended test

**Comment by Tulean** [comment]
extended check sonar

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR1607

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4747/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/397/ : SUCCESS in 14m 04s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4745/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6380/ : SUCCESS in 23m 13s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1109/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4746/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1109/ : SUCCESS in 29m 52s

File: libs/sdk-backend-spi/src/workspace/execution/index.ts [hunk 1]
```diff
@@ -151,6 +152,12 @@ export interface IPreparedExecution {
      */
     execute(): Promise<IExecutionResult>;
 
+    /**
+     * Triggers download of debugging files. If explain type is not specified, all files are included
+     * into a zip file.
+     */
+    withDebugAfm(debugConfig?: IDebugConfig): IPreparedExecution;
```
Comment (no23reason) [inline libs/sdk-backend-spi/src/workspace/execution/index.ts:159]: I'm not sure about this, this means that bear backends have to support this too. Could we somehow use the tigerSpecificFucntions instead?

**Pull request closed without merging**